src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -328,6 +328,11 @@ namespace HeatBalanceManager {
 
         GetBuildingData(state, ErrorsFound); // Read building data from input file
 
+        // need surface data to be set up before these add-on can be constructed
+        Material::GetMaterialAddOnInput(state, ErrorsFound); // Read variable thermal and solar absorptance addon data
```

Comment:
Can probably just add this code to `GetMaterialData` without this separate function, no?

Reply:
good point. I'll move it into `GetMaterialData`.